### ISSUE TITLE
Error when calling using

### ISSUE THREAD

**Issue by MaximeBouton** [Shushman/LocalFunctionApproximation.jl#1]
Error when calling `using LocalFunctionApproximation`, due to [dependency on NearestNeighbors.jl fork](https://github.com/Shushman/LocalFunctionApproximation.jl/blob/master/src/local_nn_fa.jl#L1).

```
julia> using LocalFunctionApproximation
ERROR: LoadError: LoadError: UndefVarError: NNTree not defined
Stacktrace:
 [1] include_from_node1(::String) at ./loading.jl:576
 [2] include(::String) at ./sysimg.jl:14
 [3] include_from_node1(::String) at ./loading.jl:576
 [4] eval(::Module, ::Any) at ./boot.jl:235
 [5] _require(::Symbol) at ./loading.jl:490
 [6] require(::Symbol) at ./loading.jl:405
while loading /mnt/c/Users/Maxime/wsl/.julia/v0.6/LocalFunctionApproximation/src/local_nn_fa.jl, in expression starting on line 4
while loading /mnt/c/Users/Maxime/wsl/.julia/v0.6/LocalFunctionApproximation/src/LocalFunctionApproximation.jl, in expression starting on line 71
```

**Comment by Shushman** [comment]
Yeah! I did get [this](https://github.com/KristofferC/NearestNeighbors.jl/pull/64) PR merged into `NearestNeighbors.jl` so it should be fine on the master branch now but that still won't be enough if one has just installed `NearestNeighbors` via `Pkg.add`.

**Comment by Shushman** [comment]
Fixed with #3
